lib/HTTP/Status.pm

```diff
@@ -121,7 +118,13 @@ die if $@;
 push(@EXPORT, "RC_MOVED_TEMPORARILY");
 
 my %compat = (
-    REQUEST_ENTITY_TOO_LARGE      => \&HTTP_PAYLOAD_TOO_LARGE,
+    I_AM_A_TEAPOT                 => sub {
+      $StatusCode{418} = "I'm a teapot";
```

Comment:
This is a really terrible idea. Changing global behavior based on a constant access should never happen. This should just return 418 and do nothing else.

The prototype also needs to be `()`.

Reply:
it is a terrible idea, but so is it to just change the registry.

I would like to avoid the surprise:
`say status_message HTTP_I_AM_A_TEAPOT; # (Unused)`

Reply:
If that is a large concern, just leave the message as "I'm a teapot".

The likelihood of anyone actually relying on 418 is very small.

Reply:
We use it in Production, and I often use it when developers can't get an agreement on the exact 4xx status

Reply:
And you are relying on what the message is?

There's just no way you can have this both ways. The message should either be `(Unused)` or `I'm a teapot`. I don't think it's particularly important which one. You can't do both. Trying to do both will make the result unpredictable and anything that relies on it will always be broken in some cases.

Reply:
If we need to decide on one and the status code is unused anyway, I'm in favour of 🫖.